### PR TITLE
Create Elderly Accounts

### PR DESCRIPTION
**Pull request by nouradeen**
- Create Elderly Accounts from the Caregiver account is DONE. When creating there is a field in the elderly account that references to the caregiver account's uid called **parentUID** in the database.
- Added the `CustomAppbar()` created by Juho
- And minor fixes such as 'validators' in input fields, etc...

NOTE: The caregiver can only add new accounts, I did not implement the **edit** functionality.

### CODE REVIEW

**Comment by nouradeen** [comment]
- Fixed creating Elderly accounts. A problem was when creating accounts the app force signing out the current user and logging into the newly created one. Fixed by creating a new instance of `Firebase.initializeApp()` used only when creating new accounts.
- Fixed Mohammad's caregiver Dashboard. It did not grab the data in real-time from the database. 

NOTE:
**Use `StreamBuilder()` instead of `Future()` to fetch the data in real-time**

File: lib/pages/caregiver/settings.dart [hunk 1]
```diff
@@ -297,16 +297,26 @@ class CaregiverSettings extends StatelessWidget {
...
+                            //TODO: DELETE THIS CODE-------------------------------------- CHANGE EMAIL NOT WORKING
                             IconButton(
                               onPressed: () {
                                 showDialog(
                                   barrierDismissible: false,
                                   context: context,
                                   builder: (context) => CustomPromptWidget(
                                     heading: "Update Email",
-                                    fields: const ["New Email"],
+                                    fields: ["Current Passwor", "New Email"],
```
Comment (Mnpn) [inline lib/pages/caregiver/settings.dart:309]: typo :-)

File: lib/pages/caregiver/settings.dart [hunk 2]
```diff
@@ -297,16 +297,26 @@ class CaregiverSettings extends StatelessWidget {
                                 ),
                               ),
                             ),
+
+                            //TODO: DELETE THIS CODE-------------------------------------- CHANGE EMAIL NOT WORKING
```
Comment (Mnpn) [inline lib/pages/caregiver/settings.dart:301]: why not just delete it instead of commenting to delete? :-)

**Review by Mnpn — APPROVED** [review]
LGTM, but shouldn't Juho's AppBar be a part of Juho's PR?

File: lib/pages/caregiver/settings.dart (hunk 1 above)
Comment (nouradeen) [reply lib/pages/caregiver/settings.dart:309]: Thx. Fixing on next pull

**Pull request merged**